citesphere/src/test/java/edu/asu/diging/citesphere/core/service/impl/AuthorityServiceTest.java

```diff
@@ -196,4 +197,31 @@ public void test_findByUri_noResults() {
         List<IAuthorityEntry> actual = managerToTest.findByUri(user, uri);
         Assert.assertTrue(actual.isEmpty());
     }
+    
+    @Test
+    public void test_create() {
+        String username = "user";
+        AuthorityEntry entry = new AuthorityEntry();
+        IUser user = new User();
+        user.setUsername(username);
+        Mockito.when(entryRepository.save(entry)).thenReturn(entry);
+
+        IAuthorityEntry actualEntry = managerToTest.create(entry, user);
+        Assert.assertEquals(username, actualEntry.getUsername());
+    }
+    
+    @Test
+    public void test_save() {
+        String id = "id";
+        String name = "name";
+        String uri = "uri";
+        AuthorityEntry entry = new AuthorityEntry();
+        entry.setId(id);
+        entry.setName(name);
+        entry.setUri(uri);
+        Mockito.when(entryRepository.save(entry)).thenReturn(entry);
+        
+        IAuthorityEntry actualEntry = managerToTest.save(entry);
+        Assert.assertEquals(entry, actualEntry);        
```

Comment:
same here, you want to check that the entry has actually been saved (save has been called).